Interfaces/Services/IRelationService.cs

```diff
@@ -0,0 +1,21 @@
+﻿using AndersenCoreApp.Infrastructure;
+using AndersenCoreApp.Models.DomainModels;
+using AndersenCoreApp.Models.ViewModels;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace AndersenCoreApp.Interfaces.Services
+{
+    public interface IRelationService : IMapperConfigurator
+    {
+        List<RelationViewModel> GetSortedAndFilteredRelations(IQueryable<Relation> list, string categoryName, string orderProperties);
+        bool CheckPostalMask(Relation relation);
+        void Create(RelationViewModel relation);
+        RelationViewModel GetOne(Guid id);
+        IQueryable<Relation> GetAll();
```

Comment:
megre IQueryable<Relation> GetAll(); and List<RelationViewModel> GetSortedAndFilteredRelations(IQueryable<Relation> list, string categoryName, string orderProperties);